docs/content/concepts/sui-architecture/consensus.mdx

```diff
@@ -1,109 +1,16 @@
 ---
-title: Sui Mempool and Consensus Engines
-sidebar_label: Mempool and Consensus Engines 
-description: Overview of the Sui mempool and consensus engines, Narwhal and Bullshark.
+title: Consensus
+sidebar_label: Consensus 
+description: Overview of the Sui consensus.
 ---
 
-This is a brief introduction to [Narwhal](https://github.com/MystenLabs/sui/tree/main/narwhal), and [Bullshark](https://arxiv.org/abs/2209.05633), the high-throughput mempool and consensus engines offered by Mysten Labs. Sui uses Narwhal as the mempool and Bullshark as the consensus engine by default, to sequence transactions that require a total ordering, synchronize transactions between validators and periodically checkpoint the network's state.
+## Overview
 
-The names highlight that the components split the responsibilities of:
+The basic purpose of consensus in blockchains is to agree on the order and ensure the availability of transactions.
+Validator operators and users can think of Sui consensus as a black box: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
```

Comment:
maybe `a consistent stream of deterministically ordered transactions...`

Reply:
I think consistent means the stream needs to be deterministically ordered?